src/browser/app/components/Link.react.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component, PropTypes } from 'react';
+import { Link as ReactRouterLink } from 'react-router';
+
+const styles = {
+  active: {
+    fontWeight: 'bold',
+    textDecoration: 'none',
+  },
+  hover: {
+    textDecoration: 'underline',
+  },
+  link: {
+    color: '#346392',
+    textDecoration: 'none',
+  },
+};
+
+export default class Link extends Component {
+
+  static propTypes = {
+    activeStyle: PropTypes.object,
+    children: PropTypes.node.isRequired,
+    external: PropTypes.bool,
+    style: PropTypes.object,
+  };
+
+  constructor() {
+    super();
+    this.toggleHover = this.toggleHover.bind(this);
+
+    this.state = {
+      hover: false,
+    };
+  }
+
+  toggleHover() {
+    this.setState({
+      hover: !this.state.hover,
+    });
+  }
+
+  render() {
+    const { activeStyle, children, external, style, ...otherProps } = this.props;
+    const hoverStyle = this.state.hover && styles.hover;
+
+    if (external) {
```

Comment:
I use the component for both external links (using `<a />` tags) and router-aware links (using `<Link />` from `react-router`). Right now I handle this using the `external` property.

Other way we can implement this is checking if `to` prop contains external URL or in-app link and render the `<a />` or `<Link />` accordingly.